swupd/files.go

```diff
@@ -77,11 +77,17 @@ var modifierBytes = map[ModifierFlag]byte{
 	ModifierBoot:   'b',
 }
 
-type frename bool
+// RenameFlag describes the third position in the flag string
+// and was used to represent a rename flag (deprecated) and is
+// now used to represent a mixer-generated manifest to be merged
+// with an upstream manifest via client-side mixer integration
+type RenameFlag uint8
```

Comment:
So `RenameFlag` describes something completely different to renames! This can't possibly confuse anyone!

Reply:
Yeah... this could use some clean up but I didn't want to do it in this PR.